### ISSUE TITLE
Unable to use install_modules_from_directory

### ISSUE THREAD

**Issue by glennsarti** [puppetlabs/puppet_litmus#300]
## Describe the Bug
When using the `install_modules_from_directory` rake task, it fails with the following error

```
[2] pry(main)> next
rake aborted!
Could not find metadata.json
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/puppet_litmus-0.18.2/lib/puppet_litmus/rake_helper.rb:282:in `metadata_module_name'
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/puppet_litmus-0.18.2/lib/puppet_litmus/rake_helper.rb:292:in `uninstall_module'
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/puppet_litmus-0.18.2/lib/puppet_litmus/rake_helper.rb:237:in `block in install_module'
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/honeycomb-beeline-2.0.0/lib/honeycomb/client.rb:65:in `start_span'
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/puppet_litmus-0.18.2/lib/puppet_litmus/rake_helper.rb:230:in `install_module'
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/puppet_litmus-0.18.2/lib/puppet_litmus/rake_tasks.rb:197:in `block (4 levels) in <top (required)>'
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/puppet_litmus-0.18.2/lib/puppet_litmus/rake_tasks.rb:196:in `each'
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/puppet_litmus-0.18.2/lib/puppet_litmus/rake_tasks.rb:196:in `block (3 levels) in <top (required)>'
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/puppet_litmus-0.18.2/lib/puppet_litmus/rake_tasks.rb:195:in `each'
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/puppet_litmus-0.18.2/lib/puppet_litmus/rake_tasks.rb:195:in `block (2 levels) in <top (required)>'
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/honeycomb-beeline-2.0.0/lib/honeycomb/integrations/rake.rb:21:in `block in execute'
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/honeycomb-beeline-2.0.0/lib/honeycomb/client.rb:65:in `start_span'
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/honeycomb-beeline-2.0.0/lib/honeycomb/integrations/rake.rb:16:in `execute'
C:/Source/puppet-strings/.bundle/windows/ruby/2.5.0/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
```

This is due to https://github.com/puppetlabs/puppet_litmus/commit/e60ff96b16f37a76c34a6cdc8d3df31ab0b87ecb .  https://github.com/puppetlabs/puppet_litmus/commit/e60ff96b16f37a76c34a6cdc8d3df31ab0b87ecb#diff-07f12e7d789c4559c5908eb0eb53c6fdR234 incorrectly assumes that the module to uninstall is specified in `./metadata.json`, however it could be an arbitrary module tarball that's passed into the `install_module` method. This call to uninstall_module MUST include a specific module name.

## Expected Behavior
The modules in the specified directly should be build and installed on the target node

## Steps to Reproduce
https://travis-ci.org/github/puppetlabs/puppet-strings/jobs/692825430

* Clone puppet-strings
* Make the changes to the Gemfile as per https://github.com/puppetlabs/puppet-strings/pull/236
* Run the litmus tasks as per Travis
bundle exec rake 'litmus:provision[docker, centos:7]'
bundle exec rake 'litmus:install_agent[puppet6]'
bundle exec rake 'litmus:install_modules_from_directory[./spec/fixtures/acceptance/modules]'

## Environment
 - Version Litmus 0.18.2
 - Platform Windows x64 1909

**Comment by glennsarti** [comment]
Also ping @DavidS as the ticket number is wrong on that commit  (IAC-187) doesn't exist.

**Comment by DavidS** [comment]
Thanks for the report, I'll have a look.

IAC-187 exists now.

**Comment by DavidS** [comment]
@glennsarti you have a knack for finding hairy problems :sweat_smile: - anyways. Fix for puppet-modulebuilder was released in 0.2.1 and (a series of fixes) went/is going into litmus to address this and a few adjacent niggles.

**Comment by glennsarti** [comment]
If it was easy, everyone would be on the IAC team 🥇
